generate/generate.go

```diff
@@ -846,12 +850,20 @@ func (s *service) generateNewAPICallFunc(a *API) {
 	pn("		return nil, err")
 	pn("	}")
 	pn("")
-	if n == "CreateNetwork" {
+	if n == "CreateNetwork" || n == "RegisterTemplate" {
 		pn("	if resp, err = getRawValue(resp); err != nil {")
 		pn("		return nil, err")
 		pn("	}")
 		pn("")
 	}
+	if n == "RegisterTemplate" {
+		pn("	var l []json.RawMessage")
+		pn("	if err:= json.Unmarshal(resp, &l); err != nil {")
+		pn("		return nil, err")
+		pn("	}")
+		pn("	resp = l[0]")
+		pn("")
+	}	
```

Comment:
Again some more info about what you are trying to change/solve would be nice to better understand why this would be needed...